### ISSUE TITLE
Update Bibl module entry contributor names coming from Zotero Usernames

### ISSUE THREAD

**Issue by wlpotter** [srophe/caesarea-data#51]
see, e.g., https://caesarea-maritima.org/bibl/3Q7WI48R.

"Joshua_Woods" should become "Joshua Woods"

"itsbiaancaa" should become "Bianca Gardner"

"elianayonan" should become "Eliana Yonan" (I'm not sure any of these will appear in the Bibl module as she worked primarily on the bibls for the Testimonia)

related to https://github.com/srophe/caesarea-data/issues/18

I'm not sure if we ever figured out how the best way to convert the Zotero ID to an editors.xml ID. This may need an App issue as well.

**Comment by davidamichelson** [comment]
Will add to the Zotero config file. We will need to run initiate once to redo all TEI files.

**Comment by wsalesky** [comment]
I will need to write a map in the xquery.

**Comment by wlpotter** [comment]
@wsalesky I added the table in [zotero-config.xml](https://github.com/srophe/caesarea/blob/master/zotero2bibl/zotero-config.xml)

Xpaths:

- Zotero Username ID: `zotero-config/editorsIdLookupTable/person/zoteroId`
- Editors.xml ID: `zotero-config/editorsIdLookupTable/person/sropheId` (matches `/TEI/text/body/listPerson/person/@xml:id` in editors.xml)
- Editors Name String: `zotero-config/editorsIdLookupTable/person/nameString` (corresponds to the `/TEI/teiHeader/fileDesc/titleStmt/respStmt/name` in the output TEI document)

**Comment by wlpotter** [comment]
I don't think this was implemented, see https://caesarea-maritima.vuexistapps.us/bibl/HAGQ5R8B which still has "Joshua_Woods" in the About this Online Bibliography.

**Comment by wlpotter** [comment]
The lookup table was overwritten; I will add back the lookup table and will add JLR to it as well
